
We want the non architecture specific stuff in /usr/share - not in
/usr/local/KONICA_MINOLTA/...

  Rene Rebe <rene@exactcode.de>

diff -u magicolor2430DL-1.1.0/Makefile magicolor2430DL-1.1.0-no-local/Makefile
--- magicolor2430DL-1.1.0/Makefile	2005-02-23 14:45:16.476950000 +0100
+++ magicolor2430DL-1.1.0-no-local/Makefile	2005-02-23 14:52:08.941246544 +0100
@@ -1,12 +1,13 @@
+
+MYPRINTER = /usr/share/mc2430DL
+
 CC	=	gcc
-CFLAGS	=	$(RC_CFLAGS)  -I. -I../include
+CFLAGS	=	$(RC_CFLAGS)  -I. -I../include -DKM_FOLDER=\"$(MYPRINTER)\"
 OPTIM	=	-Wall -fPIC -O2
 #OPTIM	=	-g
 LDLIBS	=	-lcups -lcupsimage -llcms -ljbig
 LDADD	=	
 
-MYPRINTER = /usr/local/KONICA_MINOLTA/mc2430DL
-
 .SILENT:
 .SUFFIXES:	.a .c .cxx .h .o .0 .1 .1m 
 .c.o:
diff -u magicolor2430DL-1.1.0/bmp.c magicolor2430DL-1.1.0-no-local/bmp.c
--- magicolor2430DL-1.1.0/bmp.c	2004-07-16 03:39:15.000000000 +0200
+++ magicolor2430DL-1.1.0-no-local/bmp.c	2005-02-23 14:49:40.056880408 +0100
@@ -159,8 +159,8 @@
 	for(plane=startplane; plane <4; plane++) //cmyk 0 1 2 3
 	{
 		sprintf(szFile, 
-			"%s/%s/Halftones/km_ht_%d%d.bmp",
-			KM_FOLDER, PRINTER_MODEL, pImage->xResolution/1200, plane); 
+			"%s/Halftones/km_ht_%d%d.bmp",
+			KM_FOLDER, pImage->xResolution/1200, plane); 
 		
 		fh = fopen( szFile, "rb");
 		if (fh == NULL)
Only in magicolor2430DL-1.1.0-no-local/: bmp.o
Common subdirectories: magicolor2430DL-1.1.0/cups and magicolor2430DL-1.1.0-no-local/cups
diff -u magicolor2430DL-1.1.0/kmlf.h magicolor2430DL-1.1.0-no-local/kmlf.h
--- magicolor2430DL-1.1.0/kmlf.h	2005-02-23 14:45:16.477950000 +0100
+++ magicolor2430DL-1.1.0-no-local/kmlf.h	2005-02-23 14:49:01.941674800 +0100
@@ -34,13 +34,6 @@
 #pragma pack()
 
 #define arch_is_big_endian __BIG_ENDIAN__
-#define PRINTER_MODEL	"mc2430DL"
-
-#ifdef __linux
-#define KM_FOLDER		"/usr/local/KONICA_MINOLTA"
-#else
-#define KM_FOLDER		"/Library/Printers/KONICA_MINOLTA"
-#endif
 
 //Media Type code
 #define PLAIN_PAPER		1
Only in magicolor2430DL-1.1.0-no-local/: rastertokmlf
diff -u magicolor2430DL-1.1.0/rastertokmlf.c magicolor2430DL-1.1.0-no-local/rastertokmlf.c
--- magicolor2430DL-1.1.0/rastertokmlf.c	2004-09-02 20:18:38.000000000 +0200
+++ magicolor2430DL-1.1.0-no-local/rastertokmlf.c	2005-02-23 14:50:31.596045264 +0100
@@ -1492,11 +1492,11 @@
 				pImage->iColorSpace = CUPS_CSPACE_RGB;
 
 				/* Set profile data */
-				//sprintf(szFile1, "%s/%s/Profiles/sRGB.icm", KM_FOLDER, PRINTER_MODEL); 
+				//sprintf(szFile1, "%s/Profiles/sRGB.icm", KM_FOLDER); 
 				//hInProfile=cmsOpenProfileFromFile(szFile1,"r");
 				hInProfile=cmsCreate_sRGBProfile();
-				sprintf(szFile2, "%s/%s/Profiles/km_%d.icm", 
-						  KM_FOLDER, PRINTER_MODEL, pImage->xResolution/1200); 
+				sprintf(szFile2, "%s/Profiles/km_%d.icm", 
+						  KM_FOLDER, pImage->xResolution/1200); 
 				hOutProfile=cmsOpenProfileFromFile(szFile2,"r");
 				hTransform = cmsCreateTransform(hInProfile,TYPE_RGB_8,hOutProfile,TYPE_CMYK_8,header.cupsRowCount,0);
 				if(err)
Only in magicolor2430DL-1.1.0-no-local/: rastertokmlf.o
